draft-ietf-rats-architecture.md

```diff
@@ -127,6 +127,88 @@ Appraisal Policy to make application-specific decisions such as authorization de
+
+Another example is the multi-chassis router which is composed of multiple single carrier-grade routers.
+The multi-chassis router provides higher throughput by interconnecting
+multiple routers and simpler management by logically seen as one router.
+Among these composing routers, there is only one main router that
+connects to the Network Management System (NMS) and the Verifier.
+Other routers are only connected to the main router by the network cables,
+and therefore they are managed and verified via this main router.
+So, in this case, the multi-chassis router is the Composite Attester,
+each router is an Attester and the main router is the Lead Attester.
+
+{{composite}} depicts the data that flows between the Composite Attester and Verifier for the remote attestation.
+
+~~~~
+                   .-----------------------------.
+                   |           Verifier          |
+                   '-----------------------------'
+                                 |  ^
+           Endorsements and      |  | Evidence of
+           Appraisal Policy for  |  | Composite
+           Evidence of Attesters |  | Attester
+                                 |  |
+.--------------------------------|--|-------------------------------.
+|                                v  |                               |
+|  .------------------------------------.                           |
+|  |  .-------------.                   |  Evidence of Attesters    |
+|  |  | Attesting   |-.                 | (via Internal Links or    |
+|  |  | Environment | |----+ Collecting |  Network Connections)     |
+|  |  '-------------' |    | Claims     |    /                      |
+|  |    '-------------'    |            |   / .------------.        |
+|  |                       v            |  /  |            |        |
+|  |                .-------------.     |<----| Attester B |--.     |
+|  |                | Target      |-.   |     '------------'  |     |
+|  |                | Environment | |   |<-------| Attester C |--.  |
+|  |                '-------------' |   |        '------------'  |  |
+|  |                  '-------------'   |<----------| ......     |  |
+|  | Lead Attester A                    |           '------------'  |
+|  '------------------------------------'                           |
+|                                                                   |
+|                       Composite Attester                          |
+'-------------------------------------------------------------------'
+~~~~
+{: #composite title="Conceptual Data Flow for Composite Attester"}
+
+In the Composite Attester, each Attester generates its own Evidence by its
+Attesting Environments collecting the Claims from its Target Environments.
+The Lead Attester collects the Evidence of all other Attesters and then
+generates the Evidence of the whole Composite Attester.
+
+After collecting the Evidence of other Attesters, the Lead Attester verifies the
```

Comment:
This paragraph as phrased doesn't make sense to me.  If it verifies evidence using policies, then it is a verifier (or a relying party), which is not what the diagram shows.  Based on the diagram, I expected this text to say that the composite attester just collects all the evidence from the individual attesters and sends it all off to the verifier (without any local verification).

Reply:
Henk said what I was trying to say. It's my mistake for the missing of the local Verifier role inside the "Lead Attester" in the diagram. My text now is about that the Lead Attester has a Verifier role inside. 

Maybe we can divide the remote attestation for the composite devices into two cases:
1. Lead Attester has no local Verifier role. The Lead Attester just collects the Evidence of other Attesters, then it integrates all these Evidence into one Evidence of the whole composite device (with no or little local processing of the collected Evidence), and it conveys the final Evidence to the Verifier.
2. Lead Attester has a local Verifier role. Then after collecting the Evidence of other attesters, the Lead Attester can verify these Evidence and integrates the verification results into the final Evidence of the whole device. This can help reduce the bandwidth for transporting the Evidence between the device and the Verifier, and also can reduce the load of the Verifier. This may be very helpful when the number of devices is large.

Reply:
Maybe we can use "Verifying Environment" to refer to the local Verifier role inside the Lead Attester.

Reply:
The Local Verifier might be a Relying Party for another verifier, it collects the Attestation Results into the Composite Evidence.